config/routes.rb

```diff
@@ -401,8 +401,8 @@
           # sometimes underscores get messed up when linked to via SMS
           as: :capture_doc_dashes
 
-      get '/in_person_proofing/state_id' => 'in_person/state_id#show'
-      put '/in_person_proofing/state_id' => 'in_person/state_id#update'
+      get '/in_person/state_id' => 'in_person/state_id#show'
+      put '/in_person/state_id' => 'in_person/state_id#update'
```

Comment:
We'll want to add some temporary redirects here to ensure that someone who's on the page can refresh or submit the form during 50/50 deployment.

```suggestion
      get '/in_person_proofing/state_id' => redirect('/in_person/state_id')
      put '/in_person_proofing/state_id' => redirect('/in_person/state_id')
      get '/in_person/state_id' => 'in_person/state_id#show'
      put '/in_person/state_id' => 'in_person/state_id#update'
```

Reply:
Ah thanks! Added those redirects. 👍

Reply:
It might be worth double-checking that this works, particularly I'm _assuming_ that a `PUT` redirect will work like that, but not 100%.

You could simulate by:

- Reaching state ID page with feature on `main`
- Switch to this branch
- Submit state ID page

Reply:
Good call to double check... I followed the pattern from the [last FSM step removal](https://github.com/18F/identity-idp/pull/9078/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5R384-R385) but I didn't think too much about the PUT

Redirecting to /verify/in_person/state_id indeed works well for GET, but the put redirect is a little funny. Here's how it looks doing what you suggested:


https://github.com/user-attachments/assets/9cd0d54e-5d7c-4d97-8eec-969913a47991


It does redirect with a PUT but doesn't submit the form successfully. It brings the user back to the current state id page with an error at the top, and their information still filled in. When they submit another time it works fine. Is this acceptable for the 50/50 state?

Reply:
It's not ideal, but I'm not sure what exactly would be involved in allowing the form to be submitted in one go.

Reply:
Yeah not ideal, and it was a simple enough fix - I made it so the put request just routes to the update action rather than being a redirect. I think this is good for now and it's temporary anyways.